src/rtc_memory_esp32.cc

```diff
@@ -131,7 +131,7 @@ static void reset_rtc(const char* reason) {
 
 // Patch the primordial entrypoint of the image (before launching FreeRTOS).
 extern "C" void IRAM_ATTR start_cpu0() {
-  ets_printf("[toit] Starting <%s>\n", toit::vm_git_version());
+  ets_printf("[toit] INFO: starting <%s>\n", toit::vm_git_version());
```

Comment:
I think this looks nicer with the other lines, so I will update Jaguar if this lands.